default/methods/source_control/file/file_browser.py

```diff
@@ -728,23 +742,68 @@ def file_view_core(
             else:
                 self.metadata['prev_page'] = None
 
-        if mode == "serialize":
-            for index_file, file in enumerate(working_dir_file_list):
-                if self.metadata['file_view_mode'] == 'explorer':
-                    file_serialized = file.serialize_with_annotations(self.session, regen_url = self.metadata["regen_url"])
+    def serialize_file_list(self, file_list):
```

Comment:
`serialize_file_list` replaces [this block](https://github.com/diffgram/diffgram/blob/38d12c685c47a53061bcbe9b80c1ae257e651a84/default/methods/source_control/file/file_browser.py#L732-L745) of code